fab/fabfile.py

```diff
@@ -638,18 +645,95 @@ def _deploy_without_asking():
         # hard update of manifest.json since we're about to force restart
         # all services
         _execute_with_timing(update_manifest)
```

Comment:
@dannyroberts @biyeun i was thinking about this some and i don't think we need to do this operation twice anymore, but i'm not quite sure. (git won't let me comment on the first instance, but i think we should get rid of that). we get Offline Compression Issues when we look for something in the manifest that doesn't exist. if we have an atomic deploy, it really shouldn't matter when the manifest.json gets written to as long as it's after we update the code and before we swap the symlink